### PR TITLE
Idfbaseline

### PR DESCRIPTION
**Pull request by gauravbaruah**
added script to generate qrels for train and dev sets. This allows for easier trec_eval

### CODE REVIEW

File: WikiQA/create-train-dev-test-data.py [hunk 1]
```diff
@@ -46,7 +48,10 @@ def write_out(infile, out_folder):
             question = ' '.join(tokenizer.tokenize(fields[1]))
             sentence = ' '.join(tokenizer.tokenize(fields[5]))
             label = fields[6]
-            qids.append(qid)
+            if qid != qidold:
+                qidold = qid
+                qidc += 1
```
Comment (rosequ) [inline WikiQA/create-train-dev-test-data.py]: @gauravbaruah rename qidc to qid_count, qidold to qid_old

File: TrecQA/build_qrels.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+for set in ['train-all', 'raw-test', 'raw-dev']:        
+    
+    with open(set + '.qrel', 'w') as qrelf:
+        labels = [y.strip() for y in open(os.path.join(set+'/sim.txt')).readlines()]
+        qids = [id.strip() for id in open(os.path.join(set+'/id.txt')).readlines()]
+
+        for i in range(len(labels)):
+            qid = qids[i]
+            lbl = labels[i]
+            ext_feats = None
```
Comment (rosequ) [inline TrecQA/build_qrels.py]: Is ext_feats needed here? if it's a no-op delete it.
I like the more pythonic way of zipping: https://docs.python.org/3/library/functions.html#zip See if you want to modify the loop to zip the files instead? 
You can refer: http://stackoverflow.com/questions/2323394/zip-alternative-for-iterating-through-two-iterables

File: WikiQA/create-train-dev-test-data.py (hunk 1 above)
Comment (gauravbaruah) [reply WikiQA/create-train-dev-test-data.py]: Fixed

File: TrecQA/build_qrels.py (hunk 2 above)
Comment (gauravbaruah) [reply TrecQA/build_qrels.py]: Fixed. zipping here may not be in convention with code elsewhere in the project that reads in labels. But good to know for the future.

**Comment by rosequ** [comment]
@lintool @gauravbaruah we can ship this now...

**Pull request merged**